shell/edit/provisioning.cattle.io.cluster/MemberRolesTab.vue

```diff
@@ -0,0 +1,53 @@
+<script>
+
+import ClusterMembershipEditor, { canViewClusterMembershipEditor } from '@shell/components/form/Members/ClusterMembershipEditor';
+import CreateEditView from '@shell/mixins/create-edit-view';
+import { Banner } from '@components/Banner';
+import Tab from '@shell/components/Tabbed/Tab';
```

Comment:
Would it not be more conveniente to keep the tabs in the `rke2.vue` file, so we do not have to import all the time and we may use the component as an abstracted view/page?

Reply:
Hi, could you please elaborate on what you mean?